Obedient-Lobsters/BangazonAPI
# Renamed appsettings and added it to gitignore and commented out httpsRedirect function for Postman

# Description
Renamed appsettings to boilerplateAppSettings added appsettings.json to .gitignore, and commented out app.UseHttpsRedirection function in Startup.cs to allow postman to work
 Fixes # (issue)
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions
Pull down this branch and check to make sure that appsettings.json is not in the branch. There should be a file titled boilerplateAppSettings.json. In Startup.cs, line 43 (containing app.UseHttpsRedirection();) should be commented out. 
 # Checklist:
